bin/common.sh

```diff
@@ -36,10 +36,6 @@ if [[ -z "${ZEPPELIN_LOG_DIR}" ]]; then
   export ZEPPELIN_LOG_DIR="${ZEPPELIN_HOME}/logs"
 fi
 
-if [[ -z "${ZEPPELIN_NOTEBOOK_DIR}" ]]; then
-  export ZEPPELIN_NOTEBOOK_DIR="${ZEPPELIN_HOME}/notebook"
-fi
```

Comment:
While `bin/zeppelin-daemon.sh` script creates notebook directory if it is not exists based on `ZEPPELIN_NOTEBOOK_DIR` env variable (see. [bin/zeppelin-daemon.sh#L97](https://github.com/apache/incubator-zeppelin/blob/master/bin/zeppelin-daemon.sh#L97)), I think directory creation should be moved from `bin/zeppelin-daemon.sh` to [VFSNotebookRepo.java#L59](https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/VFSNotebookRepo.java#L59).